### PR TITLE
Domains

### PR DESCRIPTION
**Pull request by PatAltimore**

### CODE REVIEW

File: api-reference/beta/api/domain_forcedelete.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Deletes a domain using an asynchronous operation.
+
+The following actions are performed as part of this operation: 
+* Renames the UPN, EmailAddress, and ProxyAddress of users with references to the deleted domain.
+* Renames the EmailAddress of groups with references to the deleted domain. 
+* Renames the indentifierUris of applications with references to the deleted domain. 
+* If the number of objects to be renamed is greater than 100, an error is returned. 
+* If one of the applications to be renamed is a multi-tenant app, an error is returned.
+
+After the domain deletion completes, API operations for the deleted domain will return a error responses. To verify deletion of a domain, you can perform a [get domain](domain_get.md). If the domain was successfully deleted, an error code will be returned in the response.
```
Comment (dkershaw10) [inline api-reference/beta/api/domain_forcedelete.md]: I think we can say a 404 error code will be returned.

File: api-reference/beta/api/domain_forcedelete.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# domain: forceDelete
+
+Deletes a domain using an asynchronous operation.
+
+The following actions are performed as part of this operation: 
+* Renames the UPN, EmailAddress, and ProxyAddress of users with references to the deleted domain.
+* Renames the EmailAddress of groups with references to the deleted domain. 
+* Renames the indentifierUris of applications with references to the deleted domain.
```
Comment (dkershaw10) [inline api-reference/beta/api/domain_forcedelete.md]: typo - identifierUris

File: api-reference/beta/api/domain_verify.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+POST /domains/{id}/verify
+```
+
+> For {id}, specify the domain with its fully qualified domain name.
+
+### Request headers
+
+| Name       | Description|
+|:---------------|:----------|
+| Authorization  | Bearer &lt;code&gt; Required|
```
Comment (dkershaw10) [inline api-reference/beta/api/domain_verify.md]: This should be "token" not "code".  Not sure if you have this elsewhere in this PR

File: api-reference/beta/api/domain_forcedelete.md (hunk 2 above)
Comment (PatAltimore) [reply api-reference/beta/api/domain_forcedelete.md]: Good catch. :-)

**Pull request closed without merging**